### ISSUE TITLE
0.7.1 Release

### ISSUE THREAD

**Issue by Bertram25** [OpenDungeons/OpenDungeons#849]
Here we go!! :D 

**Multiplayer:**
- [x] Lobby / Masterserver implementation #777 
  - [x] Segfaulted games are not removed from the masterserver #1040

**Creatures AI**
- [x] Add creature KO state #751 
- [x] Test and Improve if needed ranged creature behaviour #376 
- [x] Allow creatures to use skills (spells, special attacks, ...) #984 
- [x] Workers should learn to work in team. #565 

**Gameplay**
- [x] Do not show enemy creature's mood #907 
- [x] AI should handle tired/hungry creature better #909
- [x] Add the prison Room + linked logic #754 
- [x]  Add bridges #191 
- [x] ~~Add a new forge used to only create creature's equipment. #557~~
- [x] Add infinite gold tiles support #735 (Graphics are already in.)
- [x] Add prison room #754 
  - [x] Creatures in prison are free after saving/loading a game #963
- [x] Add arena room #699 
- [x] Add torture room #1058 
- [x] Add new creature Elf + DarkElf #981 

**UI**
- [x] Decide what to do about the input grabbing logic and fix it. #415
- [x] Add support for user maps + UI related changes. #383 
  - [x] ~~Editor: Add support to create a new level. #22~~ (Part 1 is done)
- [x] ~~GUI: Add copy/paste support from/to edit box (/Combo box when not read only) #675~~ 
- [x] ~~'Close' cross on research works only if clicked at its lower edge #712~~
- [x] ~~The mysterious CEGUI artefact #752~~ 
- [x] ~~Improve sliders visuals #753~~
- [x] ~~Replace project logo #1052~~
- [x] Spell cooldown counter numbers went back to jumpy #974

**Creatures**
- [x] ~~We need a researcher creature for keepers #1083~~

**Sounds**
- [x] Add heroes/Keepers voice support on certain game events. #393 

**Balancing**
- [x] ~~Ruins of the Confluent is _hard_ #1032~~
- [x] Rework Stonekeep Siege once bridges are implemented #929

**Documentation**
- [x]  ~~Add wiki doc abut how to mod creature data in level files. #854~~
- [x] Wave portals and how they work? #990
- [x] Prepare 0.7.0 announcement text + image for website.

---
- [x] Have a good lan to test multiplayer + save MP games.
- [ ] Review the CREDITS file for potential mistakes.
- [x] [CodeSpell](https://github.com/lucasdemarchi/codespell) the code.
- [ ] Review a bit the wiki overall content.
- [x] And of course, release once all that is done. ;)

**Comment by Danimal696** [comment]
Adding gem tiles

**Comment by Bertram25** [comment]
> Adding gem tiles

Ok, adding issue #735

**Comment by akien-mga** [comment]
Time to discuss what we want for this milestone \o/

**Comment by hwoarangmy** [comment]
We can already add the prison ^^

**Comment by Bertram25** [comment]
> We can already add the prison ^^

Yep, added. :)

**Comment by Bertram25** [comment]
@hwoarangmy Since you said we're starting to have enough content for another release, shoudl we start backloging issues we don't really need here. (For instance, the issues about using Tab in the GUI implies some work as a tab handler has to be implemented per window. This can for instance be moved to later.)

**Comment by hwoarangmy** [comment]
> For instance, the issues about using Tab in the GUI implies some work as a tab handler has to be implemented per window

Yes, I also had a quick look some weeks ago and it appeared to not be straight forward.

**Comment by hwoarangmy** [comment]
@Danimal696 When you will have time, it would be nice to upload elf weapons so that we can add them in game.

**Comment by Danimal696** [comment]
sure, will try today

**Comment by hwoarangmy** [comment]
@OpenDungeons/developers IMHO, we have enough stuff for a new version. I propose to start focusing on the 0.7.0 ie not working on code breaking features, planning a lan and focus on bugs fixes.

WDYT ?

**Comment by akien-mga** [comment]
I agree, there has been a ton of work done for 0.7.0 (98% by @hwoarangmy :P), we should try to make the new release. I'd propose a multiplayer game this week-end if you guys are available. Next week-end I'll be at FOSDEM so probably won't be much available (are any of you coming to FOSDEM by any chance? :D)

**Comment by Bertram25** [comment]
Arg, I wish I could come to FOSDEM. :)

I don't think I will be available this week-end as I'm working all the week-end on something else (and I'm already ill, so, need some sleep.)
But apart from that, I agree we can release and remove from the milestone everything that is a design-breaking feature.

**Comment by Bertram25** [comment]
Btw, by looking at the official list, we have a lot of bugfix pending, right?
I would move those ones to 0.8.0:
#22 (New level support) - As it is half-done and I will finish it little by little.
#557 (Forge) - Clearly game breaking feature.
#901 (New sound events) - To 0.8.0 as it really improves the game feel.
#910 (Configurable input) - As it makes the game not functional for some players. But #1041 should be dealt with, first. We just need to decide how to handle it.
#912 (Config ambient lighting) - As it improves the game feel for some players and should be rather easy.
#945 (Rooms tab) - As should be sorted out sooner than later.

I would move this to the backlog^W Later:
#675 (Copy paste support) - as it is something not urgent and I'll eventually come to it if nobody else does before me, but I don't know when, yet.
#821 (Favorite IP support) - Same reason.
#868 (Keeper hand blinking) - As we have no technical solution yet.
#900 (Separate music from SFX volume) - Same than 675

WDYT guys?

**Comment by Bertram25** [comment]
@Danimal696 @akien-mga @hwoarangmy WDYT about my proposal?

**Comment by akien-mga** [comment]
The proposal looks quite fine to me :)

**Comment by Danimal696** [comment]
i think its fine, however, i have a question. I could add in a cople of days a lot of good ambience songs; but they will need treatment, would you be willing of doing it? as an aftereffect it would also increase the game size.

**Comment by hwoarangmy** [comment]
> but they will need treatment, would you be willing of doing it?

Which treatments ?

**Comment by Danimal696** [comment]
didnt you normalize all sounds and lowered their bitrate(i think it was that)

**Comment by hwoarangmy** [comment]
I didn't. Maybe @Bertram25 or @eugeneloza

**Comment by eugeneloza** [comment]
Rready to encode/recode anything :)

**Comment by Danimal696** [comment]
Nice, ill get to upload soon :)

**Comment by hwoarangmy** [comment]
@Bertram25 ok for me

**Comment by Bertram25** [comment]
Hi guys, I've updated respective issues. :)

**Comment by akien-mga** [comment]
Hi guys, sorry for the low activity from my side, I've been quite busy with the FOSDEM and working on Godot engine (currently moving the whole wiki documentation to readthedocs.org / Sphinx, it's quite some work :D).

I'll go through the 0.7.0 issues soon too to see how we can move on. Should we plan a game sometime soon? I won't be available next week-end, but until Thursday evening this week or starting with Tuesday next week I'll be there.

**Comment by akien-mga** [comment]
@OpenDungeons/developers I've updated the first comment with a couple more issues currently assigned to the 0.7.0 milestone. We're almost there, and there are bits and pieces to work on for everyone :)

**Comment by Bertram25** [comment]
Hi guys, I'm still around even if I've got a lot to do. I'll do some code review asap, and see hat I can do to help a bit. :)

**Comment by hwoarangmy** [comment]
@OpenDungeons/developers IMHO, the only release blocker ATM is the room object for torture room. The wall objetcs are ok but not the central one.
@Danimal696 Do you have something in your cave that could fit ?

Then, we should start to release. Last release was from summer 2015 so IMHO, we should go for another one and improve what we are all working on after.

**Comment by Danimal696** [comment]
Imo we can still include a few more that are pretty basic but will add lots: new  music and the few weapons i have ready (not forge yet).
But i dont want to be the only blocker, if i cant deliver these soon, just void what i said; since lately i dont have much free time.

Ill put something together for the torture room asap.

**Comment by akien-mga** [comment]
I think we should release asap, and then start defining smaller milestones maybe, so that we can have more frequent releases. _Release early, release often_ :) The more releases we make, the more attention we might get from potential contributors. (Many people will just check when the last release was to see if a project is worth diving into, and not always when the last commits or PRs were).

**Comment by hwoarangmy** [comment]
Yes. And many people only download latest stable and stay on it. Releasing once a year is not enough IMHO. We should drive our releases dates depending on time instead of features as it would allow to have more releases.

**Comment by Bertram25** [comment]
> @OpenDungeons/developers IMHO, the only release blocker ATM is the room object for torture room. The wall objetcs are ok but not the central one.

What I'd say is, if you want to release asap, keep only bugs already in the current milestone and move everything else to later. We can decide what we'll want in 0.8 once 0.7 is out.

And then, let's focus only on stability and playability.

Ok for you guys?

**Comment by Danimal696** [comment]
Yes, there is already enought content for a release

**Comment by hwoarangmy** [comment]
@Bertram25 Do you want to let the roulette in the torture room ?

**Comment by Bertram25** [comment]
> @Bertram25 Do you want to let the roulette in the torture room ?

I don't have anything against it if you think it's eye hurting. :)

**Comment by Danimal696** [comment]
there is a replacement already

**Comment by hwoarangmy** [comment]
> there is a replacement already

Which one ? I've missed it.

**Comment by hwoarangmy** [comment]
ok, I've had a look and found it. I didn't see it before it got merged ^^

**Comment by Bertram25** [comment]
Perfect, then! :)

From what I could see, there are only two tasks left from the base plan.

**Comment by hwoarangmy** [comment]
I will try to handle #1122 tonight as it is pretty annoying and not user friendly.

**Comment by eugeneloza** [comment]
Try this one for Torture Room :)
https://www.dropbox.com/s/85zxi2argmbqwhl/160314_TortureRoom.zip?dl=0
![tortureroom1](https://cloud.githubusercontent.com/assets/10726162/13770538/73ec047a-ea90-11e5-9c53-83d7faa66cdc.png)
![tortureroom2](https://cloud.githubusercontent.com/assets/10726162/13770537/73eae018-ea90-11e5-98f9-c997f85a2e3a.png)
![tortureroom3](https://cloud.githubusercontent.com/assets/10726162/13770536/73eb0e26-ea90-11e5-8ae6-aaa4fb0cb715.png)

**Comment by Danimal696** [comment]
Those are pretty nice models :)
You still have to add shadows to it, do you know how to bake Ambient Oclussion maps?

**Comment by eugeneloza** [comment]
No :) I just add a checkbox in world -> ambient occulsion... and delete all lights to see everything properly during unwrapping the model.
I'll look through a tutorial at youtube...

**Comment by Danimal696** [comment]
you just need to overlap the AO map on the texture with an image editor, it give a nice effect

**Comment by hwoarangmy** [comment]
@Danimal696 If it is not too hard, could you please create an how to for the wiki using gimp ?

**Comment by Danimal696** [comment]
Sorry, im a photoshop user, i can do it for that program thougth

**Comment by hwoarangmy** [comment]
no pb. In this case, could you just tell what you do and post small images of an original file and the corresponding occlusion map ? I will have a look if I can find how to do it with gimp.

**Comment by Danimal696** [comment]
sure, ill do it tomorrow if possible

**Comment by hwoarangmy** [comment]
@eugeneloza The models are nice for static objects. However, most of our creature models currently have few animations: "walk", "die", "attack". If possible, we should try to focus for torture room on models for central active spots that do not make us have to add animations to all our models. The cage, for example, is good from this point of view. But the other models would need corresponding animations to be used. That's a specificity for the torture room but I guess we need to show some... torture in this room ;-) If possible funny stuff (for example a feet that would smash the creatures or stuff like that). I can handle the code parts if there is a need to resize something, to shake or anything but the animations must fit what we aim for.

**Comment by eugeneloza** [comment]
@hwoarangmy I'm afraid it's much harder for me to make animations than static models. I have almost no experience in animating, and when I do animations I get relatively random result :) And I was sure, that creatures in torture room would just walk around randomly showing "take damage" animation.
How is active spots animation is done in torture room (I haven't ever built one myself yet :D)? Do the creatures stand in the active spots and something happens to them? Or do they walk around? How are the active spots animated? By bones, precalculated animation or by flexible mesh? Can they contain particles? Is the particle system compatible with blender render or is it coded in other way?

Today I'll try to do what @Danimal696 suggested. But I have some other work to do today...

**Comment by hwoarangmy** [comment]
@eugeneloza No pb. I was only pointing this to avoid you to work on models that we won't be able to use ^^
Concerning current torture room, you can have a look by opening "model's fair" test map and droping some non worker creature in the torture room. Basically, it is the last model @Danimal696 did. Some kind of fireplace. When a creature is dropped, it goes on one acive spot and some fire (particle effect) is displayed from time to time burning the creature.

Concerning particle system, it is currently handled by code. I don't know how blender uses it nor if it is exported to ogre meshes. But if you create a model that needs particle effect, just ask and it can easily be added by code. The problem is that ATM, roomobjects are hardcoded (because really room specific with different behaviours) so adding particle effects is not straightforward.

**Comment by Danimal696** [comment]
What is the current release blocker? the cultist?

**Comment by hwoarangmy** [comment]
IMHO, there are no more release blocker. The cultist could be a nice addition but not mandatory since we already have enough creatures.
The only problem I see ATM is #1120 

In any case, I will build a release this WE with what will be in the repo and I think we can go for the 0.7.0 RC1 and a good lan would be cool to test all that. For my part, I won't add new stuff excepted maybe some unit tests or menu screen improvements. But I won't add new functionalities or code breaking stuff.

**Comment by Bertram25** [comment]
@hwoarangmy Then please, move out of the 0.7 milestone everything you don't see fit the current release so that the issue list stays relevant with what will be shipped. Good to see something out again. :)

**Comment by eugeneloza** [comment]
Short tutorial on baking Ambient Occulsion
[BakeAmbientOcculsion.zip](https://github.com/OpenDungeons/OpenDungeons/files/181920/BakeAmbientOcculsion.zip)

**Comment by eugeneloza** [comment]
And the baked textures to the models above:
[AmbientOcculsionTextures.png.zip](https://github.com/OpenDungeons/OpenDungeons/files/181927/AmbientOcculsionTextures.png.zip)
@Danimal696 , please, check if I did everything correctly.

**Comment by Danimal696** [comment]
will look tonigth :)

**Comment by Bertram25** [comment]
Based on the original plan, we only have the confluent level to update and the wiki doc to create before 0.7.0. (+the usual QA testing and website update) :)

**Comment by akien-mga** [comment]
I'll try to work on updating the release notes.

**Comment by hwoarangmy** [comment]
I've created a branch for 0.7.0 as well as windows binaries.

@akien-mga Could you please create linux binaries ?

@akien-mga @Bertram25 I let you handle the announcement part ^^

**Comment by akien-mga** [comment]
I'm currently uploading the Linux binaries. I'll try to write the release notes today.

@Bertram25 Do you want to have a go at the release announcement? You always write something nice :smiling_imp:

**Comment by Bertram25** [comment]
@akien-mga I'm back from the depth since today. I'll happily handle the announcement text and the ruin of the confluent changes tonight.
The last thing that may hurt the user experience is the wizard crazy strength balance bug IMHO. :> Too bad it was spotted, I finally had a weapon against the @hwoarangmy evil knights(TM)

**Comment by hwoarangmy** [comment]
I will fix the wizard bug soon. @Bertram25 I will wait your fix of the ruin of the confluent level before building the release then.

**Comment by Danimal696** [comment]
The cultist can already be used as well, no visible weapons though. Im uploading it today.

**Comment by akien-mga** [comment]
I got started on the changelog: https://public.etherpad-mozilla.org/p/opendungeons-0.7-changelog
I've gone through all pull requests since 0.6.0 and down to the last page of closed PRs (i.e. I still have 25 PRs to review for hidden new features that should go in the changelog). Feel free to rephrase/add missing stuff in the pad, and start thinking about what should be the highlights.

**Comment by akien-mga** [comment]
And man, this release is HUGE :D It's at least 95% @hwoarangmy's work, so I guess we'll let him pick the title for the release announcement on the blog :P

**Comment by akien-mga** [comment]
Alright, I've finished to go through all PRs, I guess the changelog is more or less complete: https://public.etherpad-mozilla.org/p/opendungeons-0.7-changelog
Just need to cherry-pick/rephrase the most awesome changes for the "Highlights" section.

**Comment by akien-mga** [comment]
@OpenDungeons/developers I cherry-picked some highlights in the pad, WDYT? If you're OK I'll make a PR.

Last blocking things for 0.7.0 IMO:
- Update CREDITS for #1115 and #1123 
- Add Cultist definitions (#1181)

**Comment by Bertram25** [comment]
@akien-mga I commented on the pad. It is really good! :) I can even on that easily for the announcement. I'll create a pad too.

**Comment by Danimal696** [comment]
Note to myself, do credits on the spot, ill get to compile them now

**Comment by akien-mga** [comment]
I think we're release-ready. Last things to do:
- Write release announcement + screenshots
- Have a multiplayer game to check for obvious issues

@OpenDungeons/developers When would you have time for a multiplayer game? I guess @hwoarangmy and I can make snapshots before that for those who can't build from source.

**Comment by hwoarangmy** [comment]
@akien-mga When do you want to play ?
@Loobinex Wanna come ?

**Comment by Loobinex** [comment]
I'm out of the country this week. Happy to try next week.

**Comment by hwoarangmy** [comment]
ok. Well, we will see when @akien-mga is available... For my part, I should be around as no holydays are planned now ^^

**Comment by akien-mga** [comment]
I'm mostly available during the week-ends, so anytime this week-end or the next would do. I can also find time during weekdays in the evening too, but maybe only for one or two games :)

**Comment by Loobinex** [comment]
I'm up for a game, but which communication channel do you use to arrange this. (I don't think setting up meetings in the comments is your preferred method). And which time zone do you use for your communication?

**Comment by hwoarangmy** [comment]
@Loobinex Usually, we connect on the OD IRC and see who is connected. Then, we start a game if enough people are around. I might try to connect tonight.

**Comment by akien-mga** [comment]
@hwoarangmy @Loobinex I'll try to be around tonight; I might only be there around 22:30 or 23:00 CEST, but feel free to play without me if I'm not there. Apparently @oyvindln would be available too.
(@Loobinex: The IRC channel is #opendungeons on Freenode)

**Comment by Danimal696** [comment]
wse are really dragging the release this time, whats the main stopper?

**Comment by Loobinex** [comment]
Well you wanted releases dates depending on time instead of features, and I figured you want exactly one year between releases.
I think for this release all that's left is playing some games to see if everything works well.

**Comment by hwoarangmy** [comment]
> Well you wanted releases dates depending on time instead of features, and I figured you want exactly one year between releases.

Originally, we wanted more something like 6 months. But we are late ^^

> wse are really dragging the release this time, whats the main stopper?

IMHO, the main stoppers we have seen while playing are fixed. Now, one or two more games would be good to check everything is ok.
We also need the announcement text for the new release

**Comment by Loobinex** [comment]
From this time forward, every summer solstice(21 June) should have a Open Dungeons release. To match the annual release on winter solstice(21 December). Celebrations will be held in your honor all around the world.

**Comment by hwoarangmy** [comment]
Then, we should hurry up ^^

**Comment by akien-mga** [comment]
We never planned releases based on time, but based on the features we want to pack in each milestone. As always, we (@hwoarangmy) have done more than planned initially, but the release is just being delayed because we're all busy and lack time to actually play-test the game before we put it out :)

**Comment by Loobinex** [comment]
On 8 March hwoarangmy proposed to time based releases, and as nobody objected, I have accepted his proposal. And set a date for the release. So you all better hurry up.
;-)

**Comment by hwoarangmy** [comment]
> We never planned releases based on time

Yes, we did but it was many time ago ;-)
Back then, we said that the list of features in the releases would be some kind of guideline but that we would try to have a release every 6 months to avoid having a last stable release very outdated.
However, we all do with the time we have and the release can wait until we can do it.

**Comment by akien-mga** [comment]
Well "trying to release every 6 months" is quite different from doing time-based releases. Time-based releases (i.e. you release on the date planned 6 months before) needs a lot more dedication than we can provide, and also has no added value for us as we don't have a big user base eagerly waiting for the new release.

So as I've likely always advocated for, "release when it's ready". We can plan the "ready" for "in approximately 6 months", but as the history shows, such a prediction will hardly be reliable. For 0.7.0, 99% of the development work was done by @hwoarangmy while @Bertram25 and I were mostly missing, busy with other stuff :)

**Comment by hwoarangmy** [comment]
> Well "trying to release every 6 months" is quite different from doing time-based releases

ok, I see what you mean. Indeed, I'm just saying that we said we would try to release every 6 months but we do what we can and if the release is not ready, it will wait.

**Comment by Bertram25** [comment]
I guess I still have to finish the announcement, right? ;)

**Comment by hwoarangmy** [comment]
It would be nice ^^

**Comment by Loobinex** [comment]
Any plans to still release any time soon, or is the new plan to keep adding issues to this milestone until you can rename it to 'version 1.0'?

**Comment by akien-mga** [comment]
It should be released asap. The main issue being to prepare a release announcement :)

**Comment by Loobinex** [comment]
If the announcement is holding it up, start freezing a version and do all new developments on the 0.8.0 milestone. With the 0.7.0 version waiting finished on a shelve the announcement will arrive sooner and doesn't have to aim for a moving target.

**Comment by hwoarangmy** [comment]
Well, the thing is that it is usually @Bertram25 who takes care of the release announcement but he doesn't seem to be around for some time. And we don't know when he will come back.

However, IMHO, we should release the 0.7.0 version anyway (even with a basic announcement) and update the text when it will be ready.

**Comment by akien-mga** [comment]
I agree; our website doesn't have much visibility either anyway, so the most important is to have a tagged version to show we're still alive IMO :)
I need to setup my chroot again to make the official binaries, but it should be easy to do.

**Comment by Loobinex** [comment]
Great. Any last minute bug fixes and/or configuration tweaks that will still go in 0.70 or do we have the final version?

**Comment by akien-mga** [comment]
I want to handle #1223 and likely #1218 tonight, but probably won't do much more for 0.7.0. I don't know if @hwoarangmy has fixes queued, but the current state is good to release IMO, so we can tag it tonight.

**Comment by hwoarangmy** [comment]
Well, I might go for the full level minimap soon but I have nothing queued at the moment.

**Comment by hwoarangmy** [comment]
IMHO, the current repo can be released.

**Comment by oyvindln** [comment]
Agreed, we should get a release out as soon as possible.

**Comment by akien-mga** [comment]
Done: https://github.com/OpenDungeons/OpenDungeons/releases/tag/0.7.0 \o/
Congrats everyone, and especially @hwoarangmy who did most of the work on this release!

I'll start working on a release announcement, if the rest of you feel like making some cool screenshots for the blog post, it would be great :)

**Comment by akien-mga** [comment]
Wow, I hadn't realized that 0.6.0 was 11 months ago already o_O

I'm currently uploading the source tarball to TuxFamily, as well as preparing the Linux binaries. (need to setup my chroot again for that)

**Comment by hwoarangmy** [comment]
windows binaries are also uploaded. Note that I won't be around for some days but I will try to come to the forum from time to time.

**Comment by Loobinex** [comment]
Congrats.

![cultists](https://cloud.githubusercontent.com/assets/13840686/17273382/506c06aa-56b3-11e6-90ba-49cf08ba4825.png)

**Comment by akien-mga** [comment]
Still WIP, but I haven't forgotten about this. I was delayed by publishing a Godot release: https://godotengine.org/article/godot-reaches-2-1-stable and now have some holidays, so I'll work on this end of August.

**Comment by hwoarangmy** [comment]
On the 0.7.0, we have found that the editor is broken. Is it worth baking 0.7.0 again ?

**Comment by akien-mga** [comment]
Now that's it's tagged, it might be better to tag 0.7.1 instead.

**Comment by hwoarangmy** [comment]
ok. Do you want to use the current dev repo or only push the needed commits ?

**Comment by akien-mga** [comment]
I think current dev would be fine, we have too little manpower to start maintaining "stable" branches (and too little players for it to matter much, I think most players will like having the very latest even if slightly less tested).

**Comment by hwoarangmy** [comment]
I agree. I will make 1 last PR for displaying the rectangle on the full minimap and then, I'm done

**Comment by akien-mga** [comment]
The good thing is that I can then skip the 0.7.0 release announcement and announce 0.7.1 directly :P

**Comment by Bertram25** [comment]
@akien-mga @hwoarangmy I've started something here: https://etherpad.net/p/od_0.7.0_release_notes (I'm trying to fulfill my promises.)
Feel free to comment and improve. A note about what's is needed next would be good to attract new people IMHO. Just my two cents, of course. ;)

**Comment by Loobinex** [comment]
So, what do you want to focus on for 0.8?

**Comment by hwoarangmy** [comment]
well, it would be 0.7.1 ^^

**Comment by Bertram25** [comment]
> well, it would be 0.7.1 ^^

I thought 0.7.1 was done? I think @Loobinex 's question is right.

**Comment by hwoarangmy** [comment]
No, we baked 0.7.0 few days ago but seen critical problems (mainly the editor that crashes at map load). Since we had merged the full drawn minimap, I just added the rectangle so the minimap can be used.

IMHO, with the PRs merged, 0.7.1 is ready to go. I will make the build ASAP.

**Comment by hwoarangmy** [comment]
I've updated the release/levels for 0.7.1

**Comment by Bertram25** [comment]
Ok, I think we're close from an actual release. :D What is left is to talk a bit about the initial plans for the 0.8.0 version, and I'll write something about that, and create a PR for the website once the release binaries are online.

**Comment by Loobinex** [comment]
That was my question, what is the focus/theme/key feature of version 0.8?

**Comment by hwoarangmy** [comment]
Binaries are uploaded:
http://download.tuxfamily.org/opendungeons/0.7.0/OpenDungeons-0.7.1-Win32-MSVS2013.zip

Only missing linux binaries

**Comment by Bertram25** [comment]
> That was my question, what is the focus/theme/key feature of version 0.8?

I was far away for quite long, so I must say I can't tell like that. @akien-mga @hwoarangmy @Danimal696 ?

**Comment by hwoarangmy** [comment]
> what is the focus/theme/key feature of version 0.8?

Well, as always, it depends on what he contributors want to do. Usually, there are many differences between what we aim for and what is done.
However, I would think that what is most needed is:
- Support win/defeat logic
- Improve creature behaviour to make the game more fun
- Use the new sound events

**Comment by Loobinex** [comment]
Setting a goal for the next milestone will help with focus and motivation, and will also help with release planning. It does not matter at all if unrelated features are worked on as well, or if focus is changed half way through.

**Comment by Bertram25** [comment]
Created PR for release notes: https://github.com/OpenDungeons/opendungeons.github.io/pull/20

I created a branch on the remote for the PR so that you can fix and push there as well.

One need to upload the binaries to tuxfamily first, though. :) @akien-mga ?

**Comment by Bertram25** [comment]
Let's speak of future goals, why not in a separate article, if at all. Improvement to the gameplay is something important it seems, in any case.

**Comment by hwoarangmy** [comment]
Yes. We also need to pecisely define how we want to tune the creature behaviour and create the corresponding issues (if they are not already)

**Comment by Danimal696** [comment]
Great update in the Webpage! :)

**Comment by Loobinex** [comment]
People should be flooding in now: https://keeperklan.com/threads/896-OpenDungeons?#post50878

**Comment by hwoarangmy** [comment]
Cool. Shall we close this one ?

**Comment by akien-mga** [comment]
> Cool. Shall we close this one ?

Yes :D
